src/VS.Web.CG.Design/VS.Web.CG.Design.nuspec

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>$id$</id>
+    <version>$version$</version>
+    <authors>$authors$</authors>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>$licenseUrl$</licenseUrl>
+    <projectUrl>$projectUrl$</projectUrl>
+    <iconUrl>$iconUrl$</iconUrl>
+    <description>$description$</description>
+    <copyright>$copyright$</copyright>
+    <tags>$tags$</tags>
+    <repository type="git" url="$repositoryUrl$" />
```

Comment:
Add commit

Reply:
@natemcmaster, when I add that and do `dotnet pack` it fails because $RepositoryCommit is not set. I looked at the nupkgs being created without my changes and they still didn't have the commit attribute in repository. Am I missing something here?

Reply:
Repository commit info gets baked into the special package metadata, not the nuspec itself (for reasons I don't understand.)

Does the package reference Internal.AspNetCore.Sdk? This is the package that has targets which set RepositoryCommit.